PIVX-Labs/MyPIVXWallet
# Add ledger persistance

## Abstract

This is a QOL feature for ledger users. It remembers the ledger account upon when logging in, so they don't have to plug in the ledger each time.
It also gives them access to contacts and easier masternode management.

---

## Testing
- Log in with ledger
- Refresh the page and check that you can still use the wallet for stuff that doesn't need spending authority

Closes #270
